src/lib/data/audio.ts

```diff
@@ -274,16 +278,30 @@ export async function getAudioSourceInfo(item: {
     };
 }
 
-//todo implement selectable text that can play audio
-
-// export function onClickSound(e: any) {
-//     let target = e.target;
-
-//     if (isSelectableText(target)) {
-//         const start = parseFloat(target.start);
-//         this.audio.currentTime = start;
-//         this.audio.play();
-//     } else {
-
-//     }
-// }
+// changes audio to the verse number clicked on
+export function verseClickSoundChange(element) {
```

Comment:
The code dealing with the html document should be in ScriptureViewSofira.svelte.  The code in audio.ts should just deal with the audio player.